trustpinning/certs.go

```diff
@@ -266,3 +284,24 @@ func parseAllCerts(signedRoot *data.SignedRoot) (map[string]*x509.Certificate, m
 
 	return leafCerts, intCerts
 }
+
+func checkCertExpiry(cert *x509.Certificate) error {
```

Comment:
Yeah, BYO-CA might warrant having different checks but I definitely agree that we should consolidate these checks into one place -- I'll update this 👍